server/tests/database/test_meeting_data_handler.py

```
@@ -111,6 +111,17 @@ def test_get_meeting_with_participants_names(app):
     assert "Bob Jones" in participants
     assert "Dave Wilson" in participants
 
+
+def test_get_meeting_users_returns_all_users(app):
```

Ruff (0.15.20)

[warning] Unused function argument: `app`

(ARG001)